### PR TITLE
RWA-2634: Replication check jobs

### PR DESCRIPTION
**Pull request by rasikasri**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: apps/wa/wa-task-batch-replication-check/prod-01.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-replication-check
+spec:
+  values:
+    job:
+      schedule: "0,10,20,30,40,50 * * * *"
+      environment:
+        JOB_NAME: PERFORM_REPLICATION_CHECK
```
Comment (thomast1906) [inline apps/wa/wa-task-batch-replication-check/prod-01.yaml:10]: you could remove duplication of environment.JOB_NAMe by moving it to its base file?  https://github.com/hmcts/cnp-flux-config/blob/88a538dbd394b0cf7126cb919bc452bc962961bd/apps/wa/wa-task-batch-replication-check/wa-task-batch-replication-check.yaml

Comment (raghera) [reply apps/wa/wa-task-batch-replication-check/prod-01.yaml:10]: Hi @thomast1906 - not sure if @rasikasri is still available or has access as he is leaving.  Do you mean adding to the base kustomization.yaml file?

The JOB_NAME is used in various different CronJobs we have which have different names so if I understand correctly the above would still have to be overridden here even if we put a default in the base kustomization.

Comment (raghera) [reply apps/wa/wa-task-batch-replication-check/prod-01.yaml:10]: Actually ignore that I guess you mean this base file: wa-task-batch-replication-check.yaml
and adding:

```
      environment:
        JOB_NAME: PERFORM_REPLICATION_CHECK
```

**Review by raghera — APPROVED** [review]

File: apps/wa/wa-task-batch-replication-check/prod-01.yaml (hunk 1 above)
Comment (raghera) [reply apps/wa/wa-task-batch-replication-check/prod-01.yaml]: Done in the latest commit

**Review by raghera — APPROVED** [review]

**Review by jordankainos — APPROVED** [review]
lgtm

**Pull request closed without merging**